File: AtlantAng/atlant-ang/src/app/entityModels/Detail.ts
import { Storekeeper } from "./Storekeeper";

export class Detail{
    id?:number;
    nomenclCode?:string;
    detailName?:string;
    detailCount?:number;
    storekeeperId?:number;
    dateOfCreation?:Date;
    dateOfRemoving?:Date;

    storekeeper?:Storekeeper;

}

File: AtlantTest/AtlantTest/DB/Entities/Detail.cs
namespace AtlantTest.DB.Entities
{
    public class Detail
    {
        public int Id { get; set; }
        public string NomenclCode { get; set; }
        public string DetailName { get; set; }
        public int DetailCount { get; set; }
        public int StorekeeperId { get; set; }
        public DateTime DateOfCreation { get; set; }
        public System.Nullable<DateTime> DateOfRemoving { get; set; }
        public Storekeeper? Storekeeper { get; set; }
    }
}
